packages/component/src/demo/EmbeddingViewMosaicDemo.svelte

```diff
@@ -42,11 +43,10 @@
     await coordinator().exec(`DROP TABLE IF EXISTS data_table`);
 
     let dataset = generateSampleDataset({ numPoints: 500000, numCategories: 3, numSubClusters: 32 });
-    let db = await coordinator().databaseConnector().getDuckDB();
     await db.registerFileText("rows.json", JSON.stringify(dataset));
     await (await db.connect()).insertJSONFromPath("rows.json", { name: "data_table" });
 
-    let r = await coordinator().query("SELECT COUNT(*) AS count FROM data_table");
+    let r: any = await coordinator().query("SELECT COUNT(*) AS count FROM data_table");
```

Comment:
Should be fixed by https://github.com/uwdata/mosaic/pull/864, but using `: any` here (any many similar cases) temporarily to avoid type check errors.